libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -585,6 +585,13 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         cmd.p1 = packet.param1;                         // minimum pitch (plane only)
         break;
 
+    case MAV_CMD_NAV_INTO_WIND:                         // MAV ID: 26
+        cmd.p1 = packet.param1;                         //distance(radius) from next waypoint
+        cmd.content.wind.deg_start = packet.param2;     //allowed angle - start of interval (0=north, 90=east)
+        cmd.content.wind.deg_stop = packet.param3;      //allowed angle - stop of interval (0=north, 90=east)
+        cmd.content.wind.altitude = packet.param4;      //altitude over the next waypoint
```

Comment:
Altitude should be stored in param7 as is the convention for altitudes on all other waypoints. It would also be more consistent to store it in cm's in the command rather then as meters. Allows a bit more percision on the altitude, and matches the rest of the behaviour in the code base. (You would need to change the definition of the altitude member of  Land_Wind_Command to be int32_t alt:24; ) But it needs to be a signed integer anyways, as a negative relative altitude is acceptable.

On that note is this only meant to support relative altitude frames? I don't see any handling of any of the different altitude frames that we support. This won'y work with MAV_FRAME_GLOBAL, MAV_FRAME_MISSION or MAV_FRAME_GLOBAL_TERRAIN_ALT.